xmrstak/jconf.cpp

```diff
@@ -118,7 +119,8 @@ xmrstak::coin_selection coins[] = {
 	{ "stellite",                {POW(cryptonight_v8_half)},   {POW(cryptonight_monero_v8)}, nullptr },
 	{ "turtlecoin",              {POW(cryptonight_turtle), 6u,POW(cryptonight_aeon)}, {POW(cryptonight_aeon)}, nullptr },
 	{ "plenteum",			     {POW(cryptonight_turtle)},    {POW(cryptonight_turtle)},    nullptr },
-	{ "zelerius",                {POW(cryptonight_v8_zelerius), 7, POW(cryptonight_monero_v8)},   {POW(cryptonight_monero_v8)}, nullptr }
+	{ "zelerius",                {POW(cryptonight_v8_zelerius), 7, POW(cryptonight_monero_v8)},   {POW(cryptonight_monero_v8)}, nullptr },
+	{ "reverse_waltz",           {POW(cryptonight_v8_reversewaltz)}, {POW(cryptonight_monero_v8)}, "mining.grftpool.com:3333" }
```

Comment:
please rename `reverse_waltz` to `cryptonight_v8_reversewaltz` and put it into alphabetic order. `cryptonight_v8_reversewaltz` is the name yo asl oput in to README.txt

Reply:
Please also remove the inactive pool.

Reply:
- `cryptonight_v8_reversewaltz` is already present, moved it into alphabetic order.
- removed `reverse_waltz`. I added it as another alias, however, if I correctly understood, you add as additional aliases only coin names, so I removed it. 
- moved `cryptonight_v8_reversewaltz` in README.txt into alphabetic order. Sorry, I didn't notice it before.
Thanks.